tests/integration/issues/list.rs

```diff
@@ -17,3 +19,32 @@ fn doesnt_fail_with_empty_response() {
     );
     register_test("issues/issues-list-empty.trycmd");
 }
+
+#[test]
+fn display_issues() {
+    let _server = mock_endpoint(
+        EndpointOptions::new(
+            "GET",
+            "/api/0/projects/wat-org/wat-project/issues/?query=&cursor=",
+            200,
+        )
+        .with_response_file("issues/get-issues.json"),
+    );
+    register_test("issues/issues-display.trycmd");
+}
+
+#[test]
+fn display_resolved_issues() {
+    let _server = mock_endpoint(
+        EndpointOptions::new(
+            "GET",
+            "/api/0/projects/wat-org/wat-project/issues/?cursor=",
+            200,
+        )
+        .with_response_file("issues/get-issues.json")
+        .with_matcher(Matcher::PartialJson(json!({
+            "query": "is:resolved",
+        }))),
+    );
+    register_test("issues/issues-display-with-query.trycmd");
```

Comment:
Okay, but the query is not filtered and I do not know why

```diff
---- expected: stdout
++++ actual:   stdout
        1 +   INFO    2023-07-19 16:05:44.849431768 +02:00 Loaded config from [CWD]/.sentryclirc
        2 +   DEBUG   2023-07-19 16:05:44.849736671 +02:00 sentry-cli version: [VERSION], platform: "linux", architecture: "x86_64"
        3 +   INFO    2023-07-19 16:05:44.849835116 +02:00 sentry-cli was invoked with the following command line: "[CWD]/target/debug/sentry-cli" "issues" "list" "--query" "is:resolved"
        4 +   DEBUG   2023-07-19 16:05:44.850717023 +02:00 request GET http://127.0.0.1:1234/api/0/projects/wat-org/wat-project/issues/?query=is:resolved&cursor=
        5 +   DEBUG   2023-07-19 16:05:44.850790381 +02:00 using token authentication
        6 +   DEBUG   2023-07-19 16:05:44.850849782 +02:00 retry number 0, max retries: 0
        7 +   DEBUG   2023-07-19 16:05:44.859579115 +02:00 > GET /api/0/projects/wat-org/wat-project/issues/?query=is:resolved&cursor= HTTP/1.1
        8 +   DEBUG   2023-07-19 16:05:44.859636333 +02:00 > Host: 127.0.0.1:1234
        9 +   DEBUG   2023-07-19 16:05:44.859652934 +02:00 > Accept: */*
       10 +   DEBUG   2023-07-19 16:05:44.859668343 +02:00 > Connection: TE
       11 +   DEBUG   2023-07-19 16:05:44.859683101 +02:00 > TE: gzip
       12 +   DEBUG   2023-07-19 16:05:44.859699101 +02:00 > User-Agent: sentry-cli/[VERSION]
       13 +   DEBUG   2023-07-19 16:05:44.865290468 +02:00 > Authorization: Bearer lolnope***
       14 +   DEBUG   2023-07-19 16:05:44.865346623 +02:00 < HTTP/1.1 200 OK
       15 +   DEBUG   2023-07-19 16:05:44.865374055 +02:00 < connection: close
       16 +   DEBUG   2023-07-19 16:05:44.865396797 +02:00 < content-type: application/json
       17 +   DEBUG   2023-07-19 16:05:44.865418939 +02:00 < content-length: 620
       18 +   DEBUG   2023-07-19 16:05:44.865521662 +02:00 response status: 200
       19 +   DEBUG   2023-07-19 16:05:44.865587426 +02:00 body: [
...
       44 + ]
       45 +
       46 +   DEBUG   2023-07-19 16:05:44.865816316 +02:00 Response dumped to: dump/api__0__projects__wat-org__wat-project__issues.json
   1   47 | +------------+-----------+---------------------------------------------------------+-----------------------------+------------+-------+
   2   48 | | Issue ID   | Short ID  | Title                                                   | Last seen                   | Status     | Level |
   3   49 | +------------+-----------+---------------------------------------------------------+-----------------------------+------------+-------+
       50 + | 4242424243 | SEN-CLI-L | ProgrammingError: column users_user.role does not exist | 2023-07-18T00:12:01.222387Z | unresolved | error |
   4   51 | | 4242424242 | SEN-CLI-H | N+1 Query                                               | 2023-07-18T00:10:01.222387Z | resolved   | info  |
       52 + | 4242424241 | SEN-CLI-1 | NameError: name 'jobs' is not defined                   | 2023-07-18T00:00:01.222387Z | ignored    | error |
   5   53 | +------------+-----------+---------------------------------------------------------+-----------------------------+------------+-------+
       54 +   DEBUG   2023-07-19 16:05:44.866845690 +02:00 skipping update nagger because session is not attended
```

the endpoint has a query `/api/0/projects/wat-org/wat-project/issues/?query=is:resolved&cursor=` but the output is not filtered. Am I using a wrong issues-file?

Reply:
Well, it's just a mock, so whatever you put in the json will be returned, and you return this exact response.
You can make an assumption that as long as this url with query is hit, it will return correct data, as its backends responsibility to do so. If url wont be hit, test wont pass.

tl;dr - create a new mock with prefiltered data